module/modmanager/manager_test.go

```diff
@@ -42,18 +43,24 @@ func TestModManagerFunctions(t *testing.T) {
 	_, err = cfgCounter1.Validate("test", resource.APITypeComponentName)
 	test.That(t, err, test.ShouldBeNil)
 
-	// This cannot use t.TempDir() as the path it gives on MacOS exceeds module.MaxSocketAddressLength.
-	parentAddr, err := os.MkdirTemp("", "viam-test-*")
-	test.That(t, err, test.ShouldBeNil)
-	defer os.RemoveAll(parentAddr)
-	parentAddr += "/parent.sock"
+	parentAddr := filepath.Join(t.TempDir(), "parent.sock")
 
```

Comment:
Flyby. I missed this when I did https://github.com/viamrobotics/rdk/pull/2902

Might be worth it to double-check that this test passes on macOS. I think it will but I haven't tested.

Reply:
Just checked; it passes on linux but hangs on MacOS and then fails for me, I'll follow up offline.

Reply:
Moved this to https://github.com/viamrobotics/rdk/pull/3202